### ISSUE TITLE
Not pinned to the desktop

### ISSUE THREAD

**Issue by jiaotamodi** [mscrivo/OotD#110]
**Describe the bug**
OotD doesn't stay on the desktop. 

**To Reproduce**
Steps to reproduce the behavior: Open clients except Outlook and then press WIN+D. 

**Expected behavior**
OotD should pinned to the desktop and when WIN+D pressed it should stay visible.

**Screenshots**
None.

**Windows Version**
- Windows 10 Version 1803

[2018-09-25.log](https://github.com/mscrivo/OotD/files/2415260/2018-09-25.log)

**Comment by mscrivo** [comment]
Yes, I'm well aware.  Microsoft has been slowing removing any ability for programs to do this.  For Windows XP, there was a really nice hack to achieve really pinning to the desktop, then in Windows 7, they took it away, but allowed you to still show a window when showing the desktop via Win+D, but now even that no longer works it seems.  I continue to look for ways of doing this, but have found nothing good yet.  If you know of any other programs that are able to do this, point them out.  Perhaps I can learn some tricks from their code if available.
